CONTRIBUTING.md

```diff
@@ -1,62 +1,152 @@
+
+Using your preferred tool you should:
+
+1. Isolate the icon from any text or extraneous items.
+1. Change the icon's viewbox/canvas/page size to 24x24.
+1. Scaled the icon to fit the viewbox, while preserving the icon's original proportions. This means the icon should be touching at least two sides of the viewbox.
+1. Center the icon horizontally and vertically.
+1. Export the icon as an SVG.
+
+### 3. Optimize The Icon
 
-## SVG example
+All icons in Simple Icons have been optimized with the [SVGO tool](https://github.com/svg/svgo). This can be done in one of two ways:
 
-Before minification, your SVG should look follow the template below:
+* The [SVGO Command Line Tool](https://github.com/svg/svgo)
+  * Install SVGO
+    * With npm: `npm install -g svgo`
+    * With Homebrew: `brew install svgo`
+  * Run the following command `svgo -p 3 icon.svg icon.min.svg`
+* The [SVGOMG Online Tool](https://jakearchibald.github.io/svgomg/)
+  * Click "Open SVG" and select an SVG file.
+  * Set the precision to 3, leave the remaining settings at the default values.
+  * Click the download button.
+
+### 4. Annotate The Icon
+
+Each icon in Simple Icons has been annotated with a number of attributes and elements to increase accessibility. These include:
+
+* An svg element with:
+  * An aria label (Note the format).
+    * `aria-labelledby="simpleicons-adobephotoshop-icon"`
+  * An img role attribute.
+    * `role="img"`
+  * A 24x24 viewbox.
+    * `viewBox="0 0 24 24"`
+  * The svg namespace.
+    * `xmlns="http://www.w3.org/2000/svg"`
+* A title element with id attribute (Note the format).
+  * `<title id="simpleicons-adobephotoshop-icon">Adobe Photoshop icon</title>`
+
+Here is _part of_ the svg for the Adobe Photoshop icon as an example:
 
 ```svg
-<svg aria-labelledby="title" role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-  <title>[BRAND] icon</title>
-  <path [VALUE] />
-</svg>
+<svg aria-labelledby="simpleicons-adobephotoshop-icon" role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title id="simpleicons-adobephotoshop-icon">Adobe Photoshop icon</title>...</svg>
 ```
 
-While the parent `<svg>` element can sometimes contain other attributes, the attributes listed above are the only necessary ones. You can safely remove attributes like `width` and `height`, or clean them using a tool like [SVGO](https://github.com/svg/svgo) or [SVGOMG](https://jakearchibald.github.io/svgomg/).
+### 5. Check The Icon
+
+The final icon should:
 
-Here is the un-minified contents of the GitHub icon, for reference:
+* Be properly annotated [as discussed above](#4-annotate-the-icon).
+* Be monochromatic.
+  * Remove all fill colors so that icon defaults to black.
+* Be scaled to fit the viewbox, while preserving the icon's original proportions.
+  * This means the icon should be touching at least two sides of the viewbox.
+* Be vertically and horizontally centered.
+* Contain one single line with no whitespace.
```

Comment:
Maybe changed to `Be minified to a single line`

Reply:
Updated.